dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -635,14 +630,17 @@
             "allOf": [
               {
                 "$ref": "#/definitions/EntitiesDocumentResult"
+              },
+              {
+                "$ref": "#/definitions/DocumentDetectedLanguage"
               }
             ]
```

Comment:
Using "allOf" to compose a schema in items is problematic for client generators -- it creates an "anonymous model". Instead, create a named schema with the allOf and make items $ref that schema.

Reply:
@bidisha-c can you help address this for Custom NER?

Reply:
doing it.